src/helpers/testData.js

```diff
@@ -309,3 +309,36 @@ export const stackedAreaData = {
     with3Sources,
     with2Sources,
 };
+
+export const legendData = [
```

Comment:
Lets make this a function that returns this array, that way we avoid bugs due to accidental change of the test data.

Reply:
Sure!